Example/Sources/SmokeAPI/ExampleOutput.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ExampleOutput.swift
+//  
+//
+//  Created by Alex Liu on 2018-10-24.
+//
```

Comment:
Would you be able to remove the `created by` part of the header and add the license header that is on other files.

Reply:
Thanks @tachyonics for your review.
Updated my file header.

Reply:
hmm, somehow the latest changes are not showing up in this existing merge request.

Reply:
Sorry for not getting back sooner. I am looking at creating a separate examples package with an example that hows a REST-like interface and using some of the other smoke-* packages. Is it alright if I resolve this pull request in favour of that work?

Reply:
Resolving this as mentioned it will be superseded by a separate examples package.